src/components/Schedule/Timeline.js

```diff
@@ -53,6 +54,22 @@ const CurrentTime = ({ start, duration, numCols }) => {
     }
   }, [])
 
+  // Scroll to where the current time indicator is
+  // I use a timeout because for some reason window initially thinks its height is short, so
+  // the scroll maxes out at this point. Obviously we don't want to, so a small timeout here.
+  useEffect(() => {
+    if (!scrolled && hoursBetweenNowAndStart > 1) {
+      const timeout = setTimeout(() => {
+        const scrollHeight = 72 + hoursBetweenNowAndStart * HOUR_HEIGHT
```

Comment:
what is the 72 here?

Reply:
This was the margin + padding of the page container and the schedule container. Just added this so that the current time indicator isn't right at the top of the screen but has some space!